src/swell/test/suite_tests/localensembleda-tier1.yaml

```
@@ -18,12 +18,13 @@ models:
     local_ensemble_save_posterior_mean: false
     local_ensemble_save_posterior_ensemble_increments: true
     ensmean_only: false
+    path_to_ensemble: /discover/nobackup/rtodling/SwellExperiments/R2D2DataStore/Shared/geos/atmens/c90/Y%Y/M%m/D%d/H%H/geos*%Y%m%d_%H%M%Sz.nc4
```

I moved those files to the test location, so just a minor change here, we use a certaun resolution nomenclature (c90 -> 91, c12 -> 13) so let's stick with that for now:

```suggestion
    path_to_ensemble: /discover/nobackup/projects/gmao/advda/SwellTestData/letk/ensemble/91/Y%Y/M%m/D%d/H%H/geos*%Y%m%d_%H%M%Sz.nc4
```

---

Resolved